From d6ffe34dc092926fb949449c182e8f27722b6b8b Mon Sep 17 00:00:00 2001
From: Dan Winship <danw@gnome.org>
Date: Mon, 29 Jul 2013 13:24:15 +0000
Subject: Update .desktop files

Don't autostart nm-applet in GNOME any more.

Update bugzilla info for new components

https://bugzilla.gnome.org/show_bug.cgi?id=705074
---
(limited to 'nm-applet.desktop.in')

--- a/nm-applet.desktop.in
+++ b/nm-applet.desktop.in
@@ -11,3 +11,4 @@
 X-GNOME-Bugzilla-Product=NetworkManager
 X-GNOME-Bugzilla-Component=nm-applet
 X-GNOME-UsesNotifications=true
+AutostartCondition=GNOME3 unless-session gnome
